### DISCUSSION TITLE
Problems with strongswan as a client on windows

### DISCUSSION THREAD

**Discussion by yw1573** [strongswan/strongswan#1400] [Q&A] [answered]
Hello, as the windows vpn client is not flexible enough, I plan to use strongswan as the windows client. I found some information on the Internet. The application of strongswan on windows is only server-side, it does not support virtual ip, so it cannot be used as a client. This blog is a long time ago and I was wondering if the current strongswan supports client features on windows

**Comment by tobiasbrunner** [comment, marked as answer]
No, virtual IPs are still not supported (see [here](https://docs.strongswan.org/docs/5.9/os/windows.html)).
